scaffolder-templates/github-workflows/basic-workflow/docs/GithubBasicDoc.md

```diff
@@ -0,0 +1,112 @@
+# Creating a Basic Serverless Workflow via a Software Template with GitHub
+
+## Overview
+This template allows the user to generate a basic workflow project from the user's input.
+The workflow will be generated corresponding to the workflow type (assessment, infrastructure) the user selects.
```

Comment:
suggestion:
> The workflow will be generated corresponding to the workflow type (assessment, infrastructure) the user selects.

Two type of workflows can be generated depending on the  workflow type inputted by the user: